### DISCUSSION TITLE
Page built failed

### DISCUSSION THREAD

**Discussion by scao53** [community/community#108314] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I've had an issue that [about.md](https://github.com/scao53/scao53.github.io/tree/master/_pages) failed to be updated. I had been able to build pages in this repo by last week. I tried multiple times yesterday and today to update [about.md](https://github.com/scao53/scao53.github.io/tree/master/_pages) but they all failed. Any suggestions and help on this would be greatly appreciated. Thanks!

**Comment by yoannchaudet** [comment]
I see you are using excerpt which is why the build was failing.

We made an update that included a bug with excerpt. This has been fixed [a few hours ago](https://github.com/jekyll/jekyll/issues/9544#issuecomment-1945281197).

A rebuild should be sufficient to get your site to build again!

**Reply by scao53** [reply]
Thanks! It works now!
